libr/anal/p/anal_arc.c

```diff
@@ -1092,6 +1092,6 @@ struct r_anal_plugin_t r_anal_plugin_arc = {
 struct r_lib_struct_t radare_plugin = {
 	.type = R_LIB_TYPE_ANAL,
 	.data = &r_anal_plugin_arc,
-	.version = R2_VERSION
+	.version = R2_VERSION,
```

Comment:
Oh ok. Hopefully no compiler will complain about this

> On 18 May 2016, at 09:19, Hamish Coleman <notifications@github.com> wrote:
> 
> In libr/anal/p/anal_arc.c:
> 
> > @@ -1092,6 +1092,6 @@ struct r_anal_plugin_t r_anal_plugin_arc = {
> >  struct r_lib_struct_t radare_plugin = {
> >  	.type = R_LIB_TYPE_ANAL,
> >  	.data = &r_anal_plugin_arc,
> > -	.version = R2_VERSION
> > +	.version = R2_VERSION,
> I added that comma so that the sys/indent works properly - without it, the indent rules yank the closing brace up onto that line instead.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>